ISAAR.MSolve.SamplesConsole/HexaSimpleCantileverBeam.cs

```diff
@@ -0,0 +1,93 @@
+﻿using ISAAR.MSolve.PreProcessor;
+using ISAAR.MSolve.PreProcessor.Elements;
+using ISAAR.MSolve.PreProcessor.Materials;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace ISAAR.MSolve.SamplesConsole
+{
+    public static class HexaSimpleCantileverBeam
+    {
+        public static void MakeCantileverBeam(Model model, double startX, double startY, double startZ, int startNodeID, int startElementID, int subdomainID)
+
+        {
+
+            int nodeID = startNodeID;
+
+            for (int j = 0; j < 4; j++)
+            {
+                if (nodeID % 2 == 0)
+                {
+                    model.NodesDictionary.Add(nodeID, new Node() { ID = nodeID, X = startX, Y = startY + 0.25, Z = startZ + 0.25 * (j / 2) });
+                }
+                else
+                {
+                    model.NodesDictionary.Add(nodeID, new Node() { ID = nodeID, X = startX, Y = startY, Z = startZ + 0.25 * (j / 2) });
+                }
+                model.NodesDictionary[nodeID].Constraints.Add(DOFType.X);
+                model.NodesDictionary[nodeID].Constraints.Add(DOFType.Y);
+                model.NodesDictionary[nodeID].Constraints.Add(DOFType.Z);
+
+                nodeID++;
+            }
+
+            for (int i = 0; i < 4; i++)
+            {
+                for (int k = 0; k < 4; k++)
+                {
+                    if (nodeID % 2 == 0)
```

Comment:
This `if...else` block can be written in one line, please revise as follows:

`model.NodesDictionary.Add(nodeID, new Node() { ID = nodeID, X = startX + 0.25 * (i + 1), Y = (nodeID % 2 == 0 ? startY + 0.25 : startY), Z = startZ + 0.25 * (k / 2) });`